include/fizzy/fizzy.h

```diff
@@ -184,20 +184,21 @@ bool fizzy_find_exported_function_index(
 /// than once with the same module results in undefined behaviour), but after fizzy_instantiate
 /// functions querying module info can still be called with @p module.
 ///
-/// @param      module                   Pointer to module. Cannot be NULL.
-/// @param      imported_functions       Pointer to the imported function array. Can be NULL iff
-///                                      imported_functions_size equals 0.
-/// @param      imported_functions_size  Size of the imported function array. Can be zero.
-/// @param      imported_table           Pointer to the imported table. Can be NULL iff module
-///                                      doesn't import a table. Not an array, because Wasm 1.0
-///                                      doesn't support more than one table in a module.
-/// @param      imported_memory          Pointer to the imported memory. Can be NULL iff module
-///                                      doesn't import a memory. Not an array, because Wasm 1.0
-///                                      doesn't support more than one memory in a module.
-/// @param      imported_globals         Pointer to the imported globals array. Can be NULL iff
-///                                      imported_globals_size equals 0.
-/// @param      imported_globals_size    Size of the imported global array. Can be zero.
-/// @returns    non-NULL pointer to instance in case of success, NULL otherwise.
+/// @param      module                     Pointer to module. Cannot be NULL.
```

Comment:
Are we also keeping some specific width between `@param` and the name? We could say that and do it in the other PR too (#528).

Reply:
I keep one space because this looks like a "command" `@param name` and this is always aligned.